### PR TITLE
style: Add some more tests for disallowed rootMargin values.

### PR DESCRIPTION
**Pull request by heycam**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/18917)

### CODE REVIEW

**Review by emilio — APPROVED** [review]

File: tests/wpt/web-platform-tests/intersection-observer/observer-exceptions.html [hunk 1]
```diff
@@ -35,6 +35,18 @@
 
 test(function () {
   assert_throws("SYNTAX_ERR", function() {
+    new IntersectionObserver(e => {}, {rootMargin: "calc(1px + 2px)"})
+  })
+}, 'IntersectionObserver constructor width { rootMargin: "calc(1px + 2px)" }');
```
Comment (emilio) [inline tests/wpt/web-platform-tests/intersection-observer/observer-exceptions.html]: nit: This should probably be `with` instead of `width`, same below.

**Comment by jdm** [comment]
Needs a `./mach update-manifest`.

**Comment by emilio** [comment]
(Just did that)

**Pull request merged**